### ISSUE TITLE
REQUEST: Create externaljwt

### ISSUE THREAD

**Issue by HarshalNeelkamal** [kubernetes/org#5224]
### New repo, staging repo, or migrate existing

staging repo

### Is it a staging repo?

yes

### Requested name for new repository

externaljwt

### Which Organization should it reside

kubernetes

### What should the repo description be?

Synced from `kubernetes/kubernetes/staging/externaljwt`, this repository contains proto APIs and plugins which enable integrating with external services for JWT signing and key management.

### What SIG and subproject does this fall under?

sig-auth

### Please provide references to appropriate approval for this new repo

- [KEP-740](https://github.com/kubernetes/enhancements/commit/8284baab861ec6efe175fabaa4f483841e137516)
- [Discussion](https://docs.google.com/document/d/1QVtBX8J2Tk70toefDegXMXigMFzrdQfezjlhBrKR29Y/edit?tab=t.0)

**Comment by liggitt** [comment]
+1 from sig-auth

(just for clarity, many of the "no response" answers are not required for staging repos)

**Comment by dims** [comment]
+1 from sig-arch

**Comment by liggitt** [comment]
any update on this? this is blocking merge of https://github.com/kubernetes/kubernetes/pull/128190

**Comment by Priyankasaggu11929** [comment]
/assign

- New repo created with a blank commit – https://github.com/kubernetes/externaljwt
- PR created to add the staging repo to k/org – https://github.com/kubernetes/org/pull/5247
- PR created to add branch protection in k/test-infra – https://github.com/kubernetes/test-infra/pull/33770

---

#### Next steps:

-  The staging repo content needs to be merged into https://github.com/kubernetes/kubernetes/tree/master/staging
(please note the repo include content from the [standard template](https://github.com/kubernetes/kubernetes-template-project), as well as, updates in the "Pull request template" to deter direct PRs)
- https://github.com/kubernetes/test-infra/pull/33770 needs to merge.
- Once all above is done, repo needs to be added to https://github.com/kubernetes/publishing-bot/blob/master/hack/repos.sh

- open question – new repo will be part of which [subproject](https://github.com/kubernetes/community/tree/master/sig-auth#subprojects) of SIG Auth?

@HarshalNeelkamal, could you take care of above steps? Thanks!

**Comment by liggitt** [comment]
> Once all above is done, repo needs to be added to https://github.com/kubernetes/publishing-bot/blob/master/hack/repos.sh

this is the only outstanding bit

the repo is part of the authentication subproject

**Comment by liggitt** [comment]
repo is created, closing this

thanks!

/close
